src/js/firstparties/google-search.js

```diff
@@ -21,6 +30,15 @@ function cleanLink(a) {
   // block event listeners on the link
   a.addEventListener("click", function (e) { e.stopImmediatePropagation(); }, true);
   a.addEventListener("mousedown", function (e) { e.stopImmediatePropagation(); }, true);
+
+  // reassign href when in firefox android
+  if (firefox_android) {
```

Comment:
Could we instead check if the link href starts with `"https://www.google.com/url?"` and contains a `'q'` search param?

Reply:
Should be able to check if the href starts with `document.location.origin` (implied I guess since it's missing in the DOM) plus `"/url?q="`.

Reply:
Good call, Added this

Reply:
Fixed nits for this piece in c58eaef99c0dd95d9081a7b2d8023bfdbf3557b8